lib/features/push_notification/presentation/controller/fcm_controller.dart

```diff
@@ -126,6 +140,10 @@ class FcmController extends BaseController {
       if (!isForeground) {
         return PushNotificationAction(typeName, newState, accountId);
       }
+    } else if (typeName == TypeName.mailboxType) {
+      if (isForeground) {
+        return SynchronizeMailboxOnForegroundAction(typeName, newState, accountId);
+      }
```

Comment:
How about the background case? MailboxView will be refreshed?

Reply:
We need to `store state change of mailbox` when we receive message from `fcm`. When the user reopens the app, we start checking for `stored state change` and then do `refresh change`